cmd/types/types.go

```diff
@@ -7,6 +7,14 @@ type UserInfo struct {
 	PublicRepos int    `json:"public_repos"`
 }
 
+//Gist contains basic info of publi gist
+type Gist struct {
```

Comment:
this struct should be singular and can be named as `gist` then have list of `gist` called `Gists`